corehq/apps/reports/filters/api.py

```diff
@@ -119,10 +128,13 @@ def options_controller(self):
 
 
 @location_safe
-class ReassignCaseOptions(CaseListFilterOptions):
+class CaseListActionOptions(CaseListFilterOptions):
     @property
     @memoized
     def options_controller(self):
+        from corehq.apps.data_interfaces.interfaces import CaseCopyInterface
```

Comment:
I wonder if we shouldn't do a copy cases privilege check here as well?

Reply:
Yeah, good idea.